core-bundle/src/Resources/contao/languages/en/tl_page.xlf

```diff
@@ -473,7 +473,7 @@
       <trans-unit id="tl_page.legacyRouting">
         <source>Disable legacy routing mode to enable this configuration.</source>
       </trans-unit>
-      <trans-unit id="tl_page.routeConflicts">
+      <trans-unit id="tl_page.routeConflicts.0">
         <source>Route conflicts</source>
       </trans-unit>
       <trans-unit id="tl_page.conflictingAlias">
```

Comment:
```suggestion
      <trans-unit id="tl_page.routeConflicts.1">
```

Should we adjust the ID here, too, then?

Reply:
Changed in d72f3aadd100b1fff875a91acad929bf4421eb5b